### DISCUSSION TITLE
github student benefits stuck in continue

### DISCUSSION THREAD

**Discussion by G-dab** [community/community#54851] [GitHub Education]
### Hi everyone,

When I was applying for github student benefits, I was stuck in continue.
I pressed 'continue' button but nothing showed up.
<img width="585" alt="image" src="https://user-images.githubusercontent.com/87303306/236808237-bfbe5624-6231-4a09-a721-c7444928414d.png">

**Comment by iamreiyn** [comment]
Hello @G-dab 

Are you able to try it on a different browser? Firefox, Edge, for example, and also from your phone if that's possible
Let us know if the problem still persists.

**Reply by G-dab** [reply]
yup I've tried Firefox and phone, but stuck when pressing 'Continue' as well

**Comment by G-dab** [comment]
found the same problem here, and it seems still unsolved yet
https://github.com/orgs/community/discussions/49378

**Comment by mishra-prajjawal** [comment]
Hello @G-dab, 
Sorry for the problem you are facing. 
There seems to be few things that could have caused the issue.
Here are some possible fixes:
1. You might have blocked the geolocation access , Till it is not given you cannot apply for GitHub Student Developer Pack.
2. Don't use a proxy (VPN), VPNs are not allowed during signup, If used can also cause rejection after a successful submission.
3. Try Clearing cookies and and cache or Start a Private Session in browser(Incognito Mode)
4. Look for any missing fields.
5. There seems to be a Javascript file missing from GitHub Education Servers but I conclude it would have been patched.
6. Change Device and look out for internet connection.
--- 
Hope it helps, If nothing works we are here to help
Thanks for contacting

**Comment by mrqsdf** [comment]
Hello, I have the same problem, but I have written above my request. the time has passed (it's been more than 1 year) but the "continue" button does not work

"Hi, mrqsdf! You were last verified as a student on May 15, 2022. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits."

**Reply by s2xon** [reply]
Same here find the solution?

**Comment by csorbakristof** [comment]
Hello! I have the same issue with teacher verification. Before requesting benefits (swag bag for students) I am redirected to verify my faculty status. There there seems to re-verify at this time (yellow note), but I cannot continue:
![image](https://github.com/community/community/assets/12133421/10e445a2-6790-4080-a715-489b094891c1)

**Reply by AshishKumar9532** [reply]
simply provide location access to the browser and reapply

**Reply by csorbakristof** [reply]
I allow all sites to ask for location, but this page does not seem to ask for it before clicking "Continue" which is disabled.
![image](https://github.com/community/community/assets/12133421/52d5c36b-bbae-4831-80b9-7093dae2734a)
(I am on Windows desktop, so as far as I know the browser application itself cannot have location permission disabled.)

**Comment by FSWei** [comment]
I solved it! I opened the settings on a Windows computer and then opened the location permission. The permission to open the browser alone is not enough

![image](https://github.com/community/community/assets/95010451/c122b662-7bc9-48f6-bcd1-6bb927478db6)

**Reply by csorbakristof** [reply]
Sounds good. I have Windows 10 and unfortunately I cannot read the screenshot above. Could you please tell me the name of those permissions? This is in the Windows level settings and not inside the browser settings, right?

**Reply by DigitAll-U** [reply]
Really useful, I've solved the issue by allowing apps access to Location in Settings as on the screen above, thanks!

**Reply by VishruthVS** [reply]
Could anyone explain similar steps in english? i couldn't understand the screenshot!!

**Reply by alex8788** [reply]
Problem solved, thank you

**Reply by RayGur** [reply]
tks, solve my problem. 
Realized how silly I was...

**Reply by dantsec** [reply]
Works to me!

**Comment by khalld** [comment]
Same problem for me, any update ? Mac OSX, tried with Safari and Chrome. Localization is enabled

**Comment by 99991** [comment]
Still broken (Firefox and Chromium, location access has been allowed). I had been verified previously in August 2022.

Edit: This specific step seems to work on Chrome in Android, but now something else is broken (GitHub complains that I should verify my email and simultaneously displays the same email address, stating that they are verified).

**Comment by LoryPack** [comment]
Broken for me as well. Please someone help

**Comment by DigitAll-U** [comment]
Just enable location in Windows Settings and browser settings, it will work
